app/commands/deliver_to_subscriber.rb

```diff
@@ -0,0 +1,18 @@
+class DeliverToSubscriber
+  attr_reader :subscriber, :email_sender
+
+  def initialize(options)
```

Comment:
Might be nicer to do `def initialize(subscriber:, email_sender:)` so it's clear what arguments are available.

Reply:
I did have it with keyword arguments (in fact I've been back and forth on it a couple of times, I've forgotten how to do new things!) but I was thinking that as the interaction with everything is currently so fluid using a hash might make it more flexible and require less changes. I also had it with `email` and `subscriber` as 'kwargs' and then `options` after that as a hash but I had an architectural crisis of confidence with that :) I'll change it now I've had a second opinion.

Reply:
Any thoughts on passing `Services.email_sender` as an argument vs just using that in the code. It's dependency injection but we already abstract it away by wrapping it in the `Services` module and we don't 'really' get any benefit in injecting it. What do you think?

Reply:
personally don't think you need to pass it in given it's going to be wrapped in the `Services`.

Reply:
Yeah, I would agree, it's probably not worth needing to pass it in.